docs/src/main/asciidoc/neo4j.adoc

```diff
@@ -4,10 +4,9 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
 = Quarkus - Neo4j
-:neo4j_version: 3.5.13
+:neo4j_version: 4.0.0
 
 include::./attributes.adoc[]
-:extension-status: preview
```

Comment:
There's still a note above the warning you removed which mentions that it is preview.